packages/new-deploy/src/bridge/BridgeContracts.ts

```diff
@@ -314,12 +329,12 @@ export default class BridgeContracts extends AbstractBridgeDeploy<config.EvmBrid
     return [];
   }
 
-  async deployCustomTokens(): Promise<void> {
+  async deployCustomTokens(): Promise<ethers.PopulatedTransaction[][]> {
```

Comment:
this is not actually called anywhere.

ideally, it would be called before relinquishing the bridge, so that in the case we're deploying a fresh network, this happens at deploy-time before relinquishing the bridge (and before any tokens are sent over the bridge)

Reply:
https://github.com/nomad-xyz/monorepo/issues/104

Reply:
flatten the `[][]` to `ethers.PopulatedTransaction[]`

Reply:
with `.flat()`

Reply:
this is not perfect cuz everywhere else deploys in one fn then enrolls in another; whereas deployCustomToken both deploys and enrolls.
but rocking with it for now: https://github.com/nomad-xyz/monorepo/pull/91/commits/4139b225aeb0bbc18ff70e49fbc10eab5ccc9cd0